addons/website_event/tests/test_website_event.py

```diff
@@ -119,3 +119,34 @@ def test_sitemap(self):
         self.assertTrue('/event/event-0' in resp.text, 'Published events must be present in the sitemap')
         self.assertTrue('/event/event-1' in resp.text, 'Published events must be present in the sitemap')
         self.assertFalse('/event/event-2' in resp.text, 'Unpublished events must not be present in the sitemap')
+
+class TestMail(MailCommon):
+
+    def test_website_publish_notification(self):
+        """ Test that the published/unpublished notifications are sent when publishing/unpublishing an event"""
+        published_subtype = self.env.ref('website_event.mt_event_published')
+        unpublished_subtype = self.env.ref('website_event.mt_event_unpublished')
+        event = self.env['event.event'].create({
+            'name': 'Event',
+            'date_begin': datetime.today() - timedelta(days=1),
+            'date_end': datetime.today() + timedelta(days=1),
+        })
+        self.flush_tracking()
+
+        follower = self.env['res.users'].create({
```

Comment:
There are already test users in mailcommon (notably a standard employee), try to reuse existing data